File: programme/length-of-string/length-of-string.c
// Write a programme to find the length of string.

#include <stdio.h>

#define LIMIT 32

int main()
{
  int length;
  char inputstr[LIMIT];

  printf("Input  : ");
  scanf("%s", inputstr);
  for (length = 0; inputstr[length] != '\0'; length++)
    ;
  printf("Output : %d\n", length);
  return 0;
}

File: programme/length-of-string/length-of-string.java
// Write a programme to find the length of string
import java.util.Scanner;

public class StringLength {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter string : ");
        String input = scanner.next();
        System.out.println("\nLength of string : " + input.length());
        scanner.close();
    }
}
